giraffe/src/types/index.ts

```diff
@@ -271,6 +272,7 @@ export type LayerConfig =
   | MosaicLayerConfig
   | TableGraphLayerConfig
   | GeoLayerConfig
+  | SimpleTableLayerConfig
```

Comment:
Same as above. Not a big deal but this should have waited until there was code that uses this definition.